src/Microsoft.ML/Models/CrossValidator.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Models
+{
+    public sealed partial class CrossValidator
+    {
+        public CrossValidationOutput<TInput, TOutput> CrossValidate<TInput, TOutput>(LearningPipeline pipeline) 
```

Comment:
>CrossValidate [](start = 54, length = 13)

Do we have a version of this for IDV as input?

Reply:
If by IDV you mean DataCollection, then yes the user can insert a data collection object in the pipeline and do CV on it.

---
In reply to: [190404310](https://github.com/dotnet/machinelearning/pull/212#discussion_r190404310) [](ancestors = 190404310)